src/Components/CreateMatchForm.js

```diff
@@ -35,13 +35,14 @@ const CreateMatchForm = ({ onSubmit }) => {
         games={games}
         name={name}
         sets={sets}
-        onIncreaseGames={() => setGames(games + 1)}
-        onDecreaseGames={() => setGames(games - 1)}
+        onIncreaseGames={() => setGames(games<7? games  + 1: games = 0)}
+        //added to prevent negative points
+        onDecreaseGames={() => setGames(games>0? games - 1: games = 7)}
         onIncreaseSets={() => setSets(sets + 1)}
-        onDecreaseSets={() => setSets(sets - 1)}
+        onDecreaseSets={() => setSets(sets>0? sets - 1: sets = 0)}
```

Comment:
You cannot assign sets = 0, because that is not how useState works. Change sets = 0 to just 0